nifi-nar-bundles/nifi-jslt-bundle/nifi-jslt-processors/src/test/java/org/apache/nifi/processors/jslt/TestJSLTTransformJSON.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.processors.jslt;
+
+
+import org.apache.nifi.util.MockFlowFile;
+import org.apache.nifi.util.TestRunner;
+import org.apache.nifi.util.TestRunners;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TestJSLTTransformJSON {
+
+    final static Path JSON_INPUT = Paths.get("src/test/resources/input.json");
+
+    @Test
+    public void testInvalidJSLTTransform() throws IOException {
+        final TestRunner runner = TestRunners.newTestRunner(new JSLTTransformJSON());
+        // This avoids an error under test where the OnScheduled method doesn't evaluate Expression Language on the transform property (after it verifies no EL is present)
+        runner.setValidateExpressionUsage(false);
+        final String inputFlowFile = "I am not JSON";
+        final String transform = new String(Files.readAllBytes(Paths.get("src/test/resources/simpleTransform.json")));
```

Comment:
Is `simpleTransform` an invalid `JSLT`? I wasn't sure when looking at it.

Reply:
It's a legit transform, it's the input that's invalid. I switched the names of the tests, will correct that.